packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -1,104 +1,141 @@
-export default {
-  builtins: {
-    Symbol: "symbol",
-    Promise: "promise",
-    Map: "map",
-    WeakMap: "weak-map",
-    Set: "set",
-    WeakSet: "weak-set",
-    setImmediate: "set-immediate",
-    clearImmediate: "clear-immediate",
-    parseFloat: "parse-float",
-    parseInt: "parse-int",
-  },
+import semver from "semver";
 
-  methods: {
-    Array: {
-      from: "array/from",
-      isArray: "array/is-array",
-      of: "array/of",
-    },
+const getDefinitions = runtimeVersion => {
+  let includeMathModule = true;
+  if (runtimeVersion) {
+    // reason for conditionally including this module:
+    // https://github.com/babel/babel/pull/8616#issuecomment-418154753
+    includeMathModule = semver.satisfies(runtimeVersion, "7.0.1 - 8.0.0");
```

Comment:
This logic isn't quite the same as what we had before. `runtimeVersion` can be a range, e.g. `^7.0.0`, and`satisfies` expects a specific version.